[2025-04-18T16:47:29.013+1000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T16:47:29.092+1000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-04-18T06:46:09.446866+00:00 [queued]>
[2025-04-18T16:47:29.141+1000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskD manual__2025-04-18T06:46:09.446866+00:00 [queued]>
[2025-04-18T16:47:29.142+1000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-18T16:47:29.264+1000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): taskD> on 2025-04-18 06:46:09.446866+00:00
[2025-04-18T16:47:29.284+1000] {standard_task_runner.py:72} INFO - Started process 85431 to run task
[2025-04-18T16:47:29.315+1000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks', 'taskD', 'manual__2025-04-18T06:46:09.446866+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/var/folders/05/dgx_ky6d3n9f_vxn26lcln1h0000gn/T/tmpzb3jhpjz']
[2025-04-18T16:47:29.333+1000] {standard_task_runner.py:105} INFO - Job 19: Subtask taskD
[2025-04-18T16:47:29.523+1000] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.taskD manual__2025-04-18T06:46:09.446866+00:00 [running]> on host anurags-macbook-pro.local
[2025-04-18T16:47:30.072+1000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anuragthakur' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T06:46:09.446866+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T06:46:09.446866+00:00'
[2025-04-18T16:47:30.077+1000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T16:47:30.309+1000] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/05/dgx_ky6d3n9f_vxn26lcln1h0000gn/T
[2025-04-18T16:47:30.316+1000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'echo TASK D completed!']
[2025-04-18T16:47:30.401+1000] {subprocess.py:99} INFO - Output:
[2025-04-18T16:47:30.431+1000] {subprocess.py:106} INFO - TASK D completed!
[2025-04-18T16:47:30.436+1000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-18T16:47:30.531+1000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T16:47:30.540+1000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskD, run_id=manual__2025-04-18T06:46:09.446866+00:00, execution_date=20250418T064609, start_date=20250418T064729, end_date=20250418T064730
[2025-04-18T16:47:30.626+1000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T16:47:30.680+1000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T16:47:30.683+1000] {local_task_job_runner.py:245} INFO - ::endgroup::
